### ISSUE TITLE
titanic.py

### ISSUE THREAD

**Issue by gindeleo** [gindeleo/machine_learning1#8]
Now you deleted the titanic.py and uploaded it again which removed everything I did :p 

Be careful with that, I don't think you should upload any files unless you add completely new files or additional non code files. Always use git to propagate changes.

If I had deleted my branch (which I could have done, since I merged it already) then whatever I have added is lost (not completely there is a backup feature on github).

I will try the merge the  files later.

**Comment by echna** [comment]
Yes, I realised. It's a bit odd. I was in my branch and uploaded the new
version, but it just puts it into the master.. So now that I have deleted a
file in my branch, but I can't just upload a new one without it ending up
in the master instead. There is no button to kill a branch either from what
I can see or am I blind? Seems like I have to just open a new branch from
the master.

Is there a way to upload a file to a branch without it ending up in the
master? Or do I just copy paste the whole text from my file into the one on
github? That seems like an odd way of doing things.

**Comment by gindeleo** [comment]
You really shouldn't upload files. You should merge or push them with git on your local machine. Maybe uploading works put 'git push' is the intended way.

You can delete a branch online once it is merged. Or locally by " git branch -d <branch_name>

**Comment by echna** [comment]
Do you use the git hub software on your machine?

**Comment by gindeleo** [comment]
Yes. No i just use git. 

Github is just a hoster that holds your files. The page let's you do some thing like editing but you should use git on your machine.

**Comment by echna** [comment]
Sorry for my incompetence. I'll do some more thesis correcting for now and
have a proper look into this stuff in the afternoon.

**Comment by gindeleo** [comment]
Don't worry no rush. We can talk tomorrow evening.
